README.md

```
@@ -12,13 +86,18 @@ erDiagram
-        INTEGER total_visitors "누적 방문자 수"
-    }
-```
\ No newline at end of file
+## Contributing
+만약 이 프로젝트에 기여하고 싶다면 언제든지 PR(Pull Request)을 보내주세요!   
+또한, Issue를 통해 버그를 제보하거나 제안을 남길 수도 있습니다. :)
+
+## Contact
+프로젝트에 대한 질문이나 제안이 있다면 아래로 연락해주세요:
+
+- Email: nbhyun0329@gmail.com
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)